src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -450,6 +454,10 @@ $pf-v5-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2
 
 // Expandable content
 .#{$toolbar}__expandable-content {
+  .#{$menu-toggle} {
+    width: 100%;
+  }
```

Comment:
Same here - can you link to where this is seen? I'm wondering if this needs to be further scoped to not do something unexpected.

Reply:
**With toggle at 100%**

![Screenshot 2023-10-05 at 1 35 10 PM](https://github.com/patternfly/patternfly/assets/5385435/4d384ea5-e2c5-4520-93fa-a6e13c3bc4e1)

Reply:
Ah, thanks! I think that should be a separate issue and we can see how to handle that. Filed an isssue here - https://github.com/patternfly/patternfly/issues/6008

I think we used to get this for free because a select was used there, and the select toggle width is 100%? I'm wondering if:
* there is ever a use case for a menu toggle that isn't a select - meaning we might not want to style all menu toggles this way
* the react examples have a fixed width assigned to the filters in the expanded content - https://www.patternfly.org/components/toolbar. Is that on purpose? If they should be full-width, should that be something the react `<Select>` component sets by default? Or should we encourage the use of `.pf-m-full-width` on the menu toggle for those filters in our examples (change the fixed width to the full width modifier)